### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by Alekuso**
**What I did:**
The changes aren't really visual but I made:
* I added prettier for tailwindcss
* Fix the PWA icons
* Switched to TS and changed script paths
* Fixed recent changes in the sveltekit api
* Made some improvements on the header, profile, and posts
* Small fixes

**Optionally: Add additional Things to this PR, or a picture of a furry animal :3**

- [X] Applied `npm run format`
- [X] Tested the changes locally

### CODE REVIEW

**Comment by VulpoTheDev** [comment]
Cute ass fox, that's a good start

File: src/routes/user/[user].svelte [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
-	export let user;
-	export let avatar;
-	export let banner;
-	export let name;
-	export let username = user;
-	export let aboutMe;
-	export let isFollowing = false;
+	let usr: string;
+
+	interface Iuser {
```
Comment (VulpoTheDev) [inline src/routes/user/[user].svelte]: Maybe change Iuser to IUser

File: src/routes/user/[user].svelte [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
 	import Post from '../../components/Post.svelte';
 
-	export let user;
-	export let avatar;
-	export let banner;
-	export let name;
-	export let username = user;
-	export let aboutMe;
-	export let isFollowing = false;
+	let usr: string;
```
Comment (VulpoTheDev) [inline src/routes/user/[user].svelte]: Change usr to user for clarity

File: src/routes/user/[user].svelte [hunk 3]
```diff
@@ -1,20 +1,32 @@
 <script context="module">
 	export async function load({ params }) {
-		let user = params.user;
-		return { props: { user } };
+		let usr = params.user;
```
Comment (VulpoTheDev) [inline src/routes/user/[user].svelte]: Change usr to user

File: src/components/Post.svelte [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
-	import { compareDates } from '../scripts/compareDates.ts';
-	export let avatar;
-	export let name;
-	export let username;
-	export let content;
+<script lang="ts">
+	import { to } from '$lib/to';
+	import { compareDates } from '$lib/compareDates';
+
+	export let avatar: string;
```
Comment (VulpoTheDev) [inline src/components/Post.svelte]: See if you can turn this into an object

**Review by VulpoTheDev — DISMISSED** [review]
Looks good, couple of changes may need to be made but either way good for production

File: src/routes/user/[user].svelte (hunk 2 above)
Comment (Alekuso) [reply src/routes/user/[user].svelte]: will change the name to `path` since it's the url string and doesn't have much of the user

**Review by VulpoTheDev — APPROVED** [review]
LGTM

**Pull request merged**